lib/main.dart

```diff
@@ -1,173 +1,378 @@
 import 'package:flutter/material.dart';
+import 'package:autocomplete_textfield/autocomplete_textfield.dart';
+import 'package:flutter/services.dart';
+import 'services/bible/korean_bible.dart';
+import 'services/bible/nkjv_bible.dart';
 
 void main() => runApp(MyApp());
 
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      title: "Interprep",
       home: Scaffold(
-        backgroundColor: Colors.grey[200],
-        body: SignUpScreen(),
-      ),
-    );
-  }
-}
-
-class SignUpScreen extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Center(
-      child: SizedBox(
-        width: 400,
-        child: Card(
-          child: SignUpForm(),
+        appBar: AppBar(
+          title: Text("Interprep"),
+          centerTitle: true,
+          backgroundColor: Colors.brown[300],
+          actions: <Widget>[
+            IconButton(
+              icon: Icon(
+                Icons.book,
+                color: Colors.white,
+              ),
+              onPressed: null,
+            ),
+          ],
         ),
+        body: CardInterface(),
       ),
+      debugShowCheckedModeBanner: false,
     );
   }
 }
 
-class SignUpForm extends StatefulWidget {
-  @override
-  _SignUpFormState createState() => _SignUpFormState();
-}
-
-class _SignUpFormState extends State<SignUpForm>
-    with SingleTickerProviderStateMixin {
-  // STEP 3: Add an AnimationController and add the
-  // AnimatedBuilder with a LinearProgressIndicator to the Column
-
-  @override
-  void initState() {
-    super.initState();
-  }
-
+class CardInterface extends StatefulWidget {
   @override
-  Widget build(BuildContext context) {
-    return Column(
-      mainAxisSize: MainAxisSize.min,
-      children: [
-        SignUpFormBody(
-          onProgressChanged: (progress) {
-            // STEP 2: Create a _formCompleted field on this class
-            // and set it here when the progress is 1
-          },
-        ),
-        Container(
-          height: 40,
-          width: double.infinity,
-          margin: EdgeInsets.all(12),
-          child: FlatButton(
-            color: Colors.blue,
-            textColor: Colors.white,
-            // STEP 1: Add a callback here and navigate to
-            // the welcome screen when the button is tapped.
-            onPressed: null,
-            child: Text('Sign up'),
-          ),
-        ),
-      ],
-    );
-  }
+  _CardInterfaceState createState() => new _CardInterfaceState();
 }
 
-class SignUpFormBody extends StatefulWidget {
-  final ValueChanged<double> onProgressChanged;
+KoreanBible koreanBible = new KoreanBible();
+NkjvBible nkjvBible = new NkjvBible();
 
-  SignUpFormBody({
-    @required this.onProgressChanged,
-  }) : assert(onProgressChanged != null);
+List<String> suggestions = koreanBible.bookNames + nkjvBible.bookNames;
```

Comment:
Hmm You don't need to instantiate KoreanBible and NkjvBible here.
You can access `KoreanBible.fullBookNames` and `NkjvBible.fullBookNames`, the static versions of the list.